packages/components/pie-lottie-player/src/defs.ts

```diff
@@ -0,0 +1,58 @@
+import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export const directions = ['forward', 'reverse'] as const;
+export type AnimationDirection = typeof directions[number];
+
+export interface LottiePlayerProps {
+    /**
+     * Lottie animation JSON file URL or relative path
+     * animationSrc and animationData are mutually exclusive
+     */
+    animationSrc?: string;
+
+    /**
+     * Object with Lottie animation data
+     * animationSrc and animationData are mutually exclusive
+     */
+    animationData?: object;
+
+    /**
+     * By the default animations loop, setting this prop as true will prevent such behaviour
+     */
+    loopDisabled?: boolean;
+
+    /**
+     * By default animations start playing as soons as its data is available, setting this prop as true will prevent such behaviour
+     */
+    autoPlayDisabled?: boolean;
```

Comment:
Personal preference but I wonder if `loop` and `autoPlay` might be easier to read here (and then set them to false instead of true). Not a big deal though!

Reply:
That was exactly the first approach I used. 

Since having both as true by default makes sense for most cases, it didn't seemed necessary to have them reflected whenever the component is initialized. Besides that would make both props required since they are booleans, and that I consider an unnecessary inconvenience for the consumer.
E.g.: 
```HTML
<pie-lottie-player animationSrc="..." loop autoplay><pie-lottie-player>
<!-- If consumer forgets to add autoplay and the animation will likely not behave as expected -->
```
Let me know if I missed something 👍

Reply:
All makes sense, thanks for the summary! 👍